src/app/contributors/contributors-loader.component.ts

```diff
@@ -0,0 +1,38 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'app-contributors-loader',
+  template: `
+    <div class="loader-container">
+      <div class="loader"></div>
+      <div class="loader"></div>
+      <div class="loader"></div>
+      <div class="loader"></div>
+      <div class="loader"></div>
```

Comment:
Suggestion: Good to make this configurable with a constant for how many we need to show via `@Input`.

I wonder if we need a dedicated loading component just for the contributors' section instead of a generic one that could be reused elsewhere.

Reply:
I agree with the general loader component but I think it would be good if we implement a general loader component in the future if we want or maybe we can use `ngx-skeleton-loader` library.